check-cla/action.yml

```diff
@@ -102,6 +105,39 @@ runs:
               name: '${{ inputs.label }}'
             })
 
+    # checkout conda/infrastructure to update .clabot
+    - uses: actions/checkout@v3
+      if: steps.contributors.outputs.hasSigned == 'false'
+      with:
+        repository: conda/infrastructure
+
+    # update .clabot
+    - shell: python
+      if: steps.contributors.outputs.hasSigned == 'false'
+      run: |
+        import json
+        from pathlib import Path
+
+        path = Path(".clabot")
```

Comment:
I wonder if it's time to rename this file to `cla/signatories.json`?

Reply:
I'm still contemplating if it makes sense to combine `.authors.yml` defined in our `rever` repos with `.clabot`, the former is just additional metadata to the latter. Either way, I'll leave this exercise for a future effort.

Reply:
Another time maybe! :)